src/ui/libs/DatalensChartkit/menu/MenuItems.tsx

```diff
@@ -240,20 +243,27 @@ export const getLinkMenuItem = (customConfig?: Partial<MenuItemConfig>): MenuIte
         customConfig?.action ||
         function action({loadedData, propsData}) {
             return function render(props: MenuItemModalProps) {
+                let initialParams = {};
+                if (!isExternal) {
```

Comment:
Very bad naming of arg

Reply:
Remove arg and changed it to feature flag, I think it will be better this way